core/src/main/java/org/apache/accumulo/core/metadata/StoredTabletFile.java

```diff
@@ -151,7 +150,11 @@ public String toString() {
    * Validates that the provided metadata string for the StoredTabletFile is valid.
    */
   public static void validate(String metadataEntry) {
+    final TabletFileCq tabletFileCq = deserialize(metadataEntry);
+    // Validate the path
     ReferencedTabletFile.parsePath(deserialize(metadataEntry).path);
+    // Validate the range
+    requireRowRange(tabletFileCq.range);
```

Comment:
`errorprone` might complain that you are not using the return value.

Reply:
I ran it against the errorprone profile and it didn't complain about it but if it breaks in the future I can fix it of course.